From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Sat, 27 Oct 2018 17:54:01 +0200
Subject: Disable update scheduler

Disable the update scheduler, WebAPK update, background sync one-shot and
notification scheduler background tasks so that the browser does not wake
up periodically in the background.
License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 .../background_task_scheduler/ChromeBackgroundTaskFactory.java  | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/background_task_scheduler/ChromeBackgroundTaskFactory.java b/chrome/android/java/src/org/chromium/chrome/browser/background_task_scheduler/ChromeBackgroundTaskFactory.java
--- a/chrome/android/java/src/org/chromium/chrome/browser/background_task_scheduler/ChromeBackgroundTaskFactory.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/background_task_scheduler/ChromeBackgroundTaskFactory.java
@@ -74,8 +74,8 @@ public class ChromeBackgroundTaskFactory implements BackgroundTaskFactory {
             case TaskIds.WEBAPK_UPDATE_JOB_ID:
-                return new WebApkUpdateTask();
+                return null;
             case TaskIds.COMPONENT_UPDATE_JOB_ID:
-                return new UpdateTask();
+                return null;
             case TaskIds.BACKGROUND_SYNC_ONE_SHOT_JOB_ID:
-                return new BackgroundSyncBackgroundTask();
+                return null;
             case TaskIds.NOTIFICATION_SCHEDULER_JOB_ID:
-                return new NotificationSchedulerTask();
+                return null;
--
